openbas-front/src/admin/components/agents/Agents.tsx

```diff
@@ -264,302 +516,179 @@ SHA512: ca07dc1d0a5297e29327e483f4f35dadb254d96a16a5c33da5ad048e6965a3863d621518
-                  width: '100%',
-                  right: 0,
-                  bottom: 10,
-                  display: 'flex',
-                  alignItems: 'center',
-                  justifyContent: 'center',
-                }}
-                >
-                  {macOsExecutors.map((executor: Executor) => {
-                    return (
-                      <img
-                        key={executor.executor_id}
-                        src={`/api/images/executors/${executor.executor_type}`}
-                        alt={executor.executor_type}
+                      >
+                        {t('Install')}
+                        {' '}
+                        {executor.executor_name}
+                      </Typography>
+                      <div
                         style={{
-                          width: 30,
-                          height: 30,
-                          borderRadius: 4,
-                          margin: '0 10px 0 10px',
+                          position: 'absolute',
+                          width: '100%',
+                          display: 'flex',
+                          alignItems: 'center',
+                          justifyContent: 'center',
                         }}
-                      />
-                    );
-                  })}
-                </div>
-              </CardContent>
-            </CardActionArea>
-          </Card>
-        </Grid>
+                      >
+                        {platforms.map(platform => (
+                          <div
+                            key={platform}
+                            style={{
+                              margin: theme.spacing(0, 1),
+                              padding: theme.spacing(1),
+                              border: '1px solid',
+                              borderRadius: '4px',
+                              borderColor: '#292D39',
+                              display: 'flex',
+                              justifyContent: 'center',
+                              alignItems: 'center',
+                            }}
+                          >
+                            <PlatformIcon platform={platform} width={20} />
+                          </div>
+                        ))}
+                      </div>
+                    </div>
+                  </CardContent>
+                </CardActionArea>
+              </Card>
+            </Grid>
+          );
+        })}
       </Grid>
       <Dialog
-        open={platform !== null}
+        open={selectedExecutor !== null}
         TransitionComponent={Transition}
         onClose={closeInstall}
-        PaperProps={{ elevation: 1 }}
-        fullWidth={true}
+        slotProps={{ paper: { elevation: 1 } }}
+        fullWidth
         maxWidth="md"
       >
-        <DialogTitle>{t('Install a simulation agent')}</DialogTitle>
+        <DialogTitle style={{ padding: theme.spacing(6, 4, 5, 5) }}>
+          {selectedExecutor?.executor_name}
```

Comment:
```suggestion
{`${selectedExecutor?.executor_name} ${t('Installation')}`}
```
Consider to use template litteral it make the code more readable and concise and it easier to maintain